torch/fx/experimental/migrate_gradual_types/z3_types.py

```diff
@@ -0,0 +1,28 @@
+try:
```

Comment:
This looks like a test. Can you move this to a file in https://github.com/pytorch/pytorch/tree/master/test/fx

It might also be good to wrap this stuff up in a `TestCase` instance to help give better interfacing to test runners and diagnostic information. See `test_dce_pass.py` for an example: https://github.com/pytorch/pytorch/blob/f749f86feec312b0efc9237d540973fa8414cd96/test/fx/test_dce_pass.py#L10